oat-sa/extension-tao-itemqti
# Fix/tca 738/theme table headers color

Related to : https://oat-sa.atlassian.net/browse/TCA-738
 
The fix of table headers color for items on white-on-dark contrast themes
  
#### How to test
 
- prepare an instance of TAO
- prepare a delivery of test with tables
- execute delivery as a test taker
- switch to contrast theme "Light yellow on royal blue"
- ensure table headers is readable
